docs/source/en/pipeline_webserver.md

```diff
@@ -87,6 +87,12 @@ of the model on the webserver. This way, no unnecessary RAM is being used.
 Then the queuing mechanism allows you to do fancy stuff like maybe accumulating a few
 items before inferring to use dynamic batching:
 
+<Tip warning={true}>
+**Important**
+The code is intentionally written like pseudo-code for readability.
+Do not run this without checking if it makes sense for your system resources!
+</Tip>
```

Comment:
No need to add "important" since this warning callout means essentially the same thing :)

```suggestion
<Tip warning={true}>

The code sample below is intentionally written like pseudo-code for readability.
Do not run this without checking if it makes sense for your system resources!

</Tip>
```